service/src/main/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsNotificationService.java

```diff
@@ -27,19 +31,38 @@
     private final EmailService emailService;
     private final RoboticsDataService roboticsDataService;
     private final RoboticsEmailConfiguration roboticsEmailConfiguration;
+    private final ObjectMapper mapper;
 
     @EventListener
     public void notifyRoboticsOfSubmittedCaseData(final CaseNumberAdded event) {
-        RoboticsData roboticsData = roboticsDataService.prepareRoboticsData(event.getCaseData());
+        sendSubmittedCaseData(event.getCaseDetails());
+    }
+
+    public void sendSubmittedCaseData(final CaseDetails caseDetails) {
```

Comment:
technically there is nothing that verify that case is submitted, it is possible that support team will try to use it to send not submitted cases